.github/workflows/compilation.yml

```diff
@@ -0,0 +1,68 @@
+name: CI-compile
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-windows:
+    name: Windows compilation
+    runs-on: windows-2019
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Setup msbuild
+        uses: microsoft/setup-msbuild@v1.1
+        # with:
+        #   vs-version: "[15.0,16.0)"
+        #   msbuild-architecture: x64
+
+      - name: Build
+        run: msbuild.exe cli\\maxcso.sln -m -p:Configuration=Release -p:PlatformToolset=v141_xp -p:Platform=x64 -p:TrackFileAccess=false
```

Comment:
Hm, did it fail without specifying PlatformToolset?  I'm gonna try without that, because I'd prefer to be able to merge #34 without changing this file.  On PPSSPP, I don't remember needing it...

-[Unknown]

Reply:
yes, it will fail, your project specs are too outdated, Travis was using msvc2017, but visual studio 2019 needs to include this flag. Or you can update your project specs by opening it in the modern visual studio. I didnt do tht cause it will raise many warning, some of them can be treated as errors, so I just provide compilable solution.

Reply:
It did build okay without it:
https://github.com/unknownbrackets/maxcso/commit/d30c9687b634d0738b656caf33d8fdb925abcafd#diff-f9d9a4b23f1d680a037541fcd155069b6c4ca2db38a49414383ab735de964688L23

I probably will drop XP support in not that long, but there hasn't really been a reason to do it other than newer Visual Studio versions dropping support.  I prefer to drop support for things in response to real limitations, not just because I think I know better than users still using XP.

-[Unknown]